### PR TITLE
Get Locations Feature

### PR DESCRIPTION
**Pull request by KristianJBorgwarth**
Added:
GetLocationsQuerySerializer
GetLocationsQuery
GetLocationsQueryHandler

- since the request results in a query, the related classes are called queries, instead of commands, and are placed in a queries/location directory. They still inherit from Request and RequestHandler

GetLocationsQueryHandler retrieves Locations using pagination to limit amount of data transfared. 

I have added some  TODO: SUGGESTIONs in Serializer classes

### CODE REVIEW

File: core/views/ResponseEnvelope.py
Comment (KristianJBorgwarth) [inline core/views/ResponseEnvelope.py:1]: I also removed the to_response and just added a return Response({}) in the fail and success of ResponseEnvelope. This fixed an issue where our intended status code didn't get returned but defaulted to 400 or 200

File: core/common/validators.py [hunk 1]
```diff
@@ -23,6 +23,15 @@ def validate_max_length(value, max_length):
         raise serializers.ValidationError(ErrorMessage.must_be_less_than(value, max_length))
 
 
+def validate_integer(value, min_value=None, max_value=None):
+    if not isinstance(value, int):
+        raise serializers.ValidationError(ErrorMessage.must_be_of_type(value, 'integer'))
```
Comment (lmschwenger) [inline core/common/validators.py:28]: we can generalize this to:

```
type = int
if not isinstance(value, type):
    raise serializers.ValidationError(ErrorMessage.must_be_of_type(value, type))
```

File: tests/unittests/location/querytest/get_locations_test/test_get_locations_query_handler.py [hunk 2]
```diff
@@ -0,0 +1,86 @@
+import os
+
+import django
+
```
Comment (lmschwenger) [inline tests/unittests/location/querytest/get_locations_test/test_get_locations_query_handler.py:4]: remove empty line, so there's only 1 line in-between

File: tests/unittests/location/querytest/get_locations_test/test_get_locations_query_serializer.py [hunk 3]
```diff
@@ -0,0 +1,39 @@
+
```
Comment (lmschwenger) [inline tests/unittests/location/querytest/get_locations_test/test_get_locations_query_serializer.py:1]: Remove empty start line

**Review by lmschwenger — APPROVED** [review]
Agree with the TODO: SUGGESTIONS. -- But maybe we should wait until we add more Classes in the selected dirs

**Pull request merged**